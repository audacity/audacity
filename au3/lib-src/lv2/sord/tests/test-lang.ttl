@prefix : <http://example.org/test-lang#> .

:thing :greeting "Hello"@en ;
       :greeting "Howdy"@en-us ;
       :greeting "Bonjour"@fr ;
       :greeting "Guten Tag"@de-latn-de ;
       :greeting "HEY MAN"@en-crazy0place .
