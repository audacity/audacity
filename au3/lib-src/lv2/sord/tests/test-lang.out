<http://example.org/test-lang#thing> <http://example.org/test-lang#greeting> "Hello"@en .
<http://example.org/test-lang#thing> <http://example.org/test-lang#greeting> "Howdy"@en-us .
<http://example.org/test-lang#thing> <http://example.org/test-lang#greeting> "Bonjour"@fr .
<http://example.org/test-lang#thing> <http://example.org/test-lang#greeting> "Guten Tag"@de-latn-de .
<http://example.org/test-lang#thing> <http://example.org/test-lang#greeting> "HEY MAN"@en-crazy0place .
